gen_mod_headers

```diff
@@ -155,6 +155,14 @@ for f in Makefile_32.cpu kernel/asm-offsets.s; do
 	[ -f $p ] && cp "$p" "$target_karch_dir/$f"
 done
 
+# Address sources with this kind of hacks
+# https://github.com/raspberrypi/linux/commit/a66649dab35033bd67988670fa60c268b0444cda
```

Comment:
Does the referenced commit fix the issue and make this change unnecessary? If so would be good to note which kernel version it is merged in.